app/Models/Role.php

```diff
@@ -96,7 +97,8 @@ class Role extends Model
         self::STUDENT_COUNCIL_SECRETARY,
         self::STUDENT_COUNCIL,
         self::BOARD_OF_TRUSTEES_MEMBER,
-        self::ETHICS_COMMISSIONER
+        self::ETHICS_COMMISSIONER,
+        self::KKT_HANDLER,
```

Comment:
Maybe this could go under `COMMITTEE_MEMBERS`; they were originally members of the Economic Committee, and they belong under the economic vice president, rather than being independent office holders. This way, it would be under the "Students Council" dropdown, which might be a more logical choice. But maybe this is only my preference:)